VultisigApp/VultisigApp/Services/Keysign/BlockChainSpecific.swift

```diff
@@ -14,10 +14,10 @@ enum BlockChainSpecific: Codable, Hashable {
     case THORChain(accountNumber: UInt64, sequence: UInt64, fee: UInt64)
     case MayaChain(accountNumber: UInt64, sequence: UInt64)
     case Cosmos(accountNumber: UInt64, sequence: UInt64, gas: UInt64)
-    case Solana(recentBlockHash: String, priorityFee: BigInt) // priority fee is in microlamports
+    case Solana(recentBlockHash: String, priorityFee: BigInt, fromAddressPubKey: String?, toAddressPubKey: String?) // priority fee is in microlamports
```

Comment:
what you think about separating Solana blockchainSpecific into: .solana, .solanaToken, .solanaCreateToken

Reply:
I believe it will bring more complexity to our code. So I would recommend us to keep it in the same service. It already handles it for Native, New Token Accounts, and Already associated Token acounts.